src/Core/Configurations/RuntimeConfigProvider.cs

```diff
@@ -83,17 +80,17 @@ public RuntimeConfig GetConfig()
         }
 
         // While loading the config file, replace all the environment variables with their values.
-        ConfigLoader.TryLoadKnownConfig(out _, replaceEnvVar: true);
+        ConfigLoader.TryLoadKnownConfig(out RuntimeConfig? runtimeConfig, replaceEnvVar: true);
 
-        if (ConfigLoader.RuntimeConfig is null)
+        if (runtimeConfig is null)
```

Comment:
why not use the output of `TryLoadKnownConfig`? that result will implicitly tell you whether out param `runtimeConfig` is null or not.

Reply:
saving 1 bit of memory on the stack of course ;)

sure Ill swap that around :D

Reply:
The actual reason was to get the compiler to be quiet about returning a possible null value, it doesnt recognize that if the return value is true here we have a non-null, but I can add the ! operator to quiet it up.